src/meta/owner.rs

```diff
@@ -37,10 +44,18 @@ impl<'a> From<&'a Metadata> for Owner {
 
 impl Owner {
     pub fn render_user(&self, colors: &Colors) -> ColoredString {
-        colors.colorize(self.user.clone(), &Elem::User)
+        if self.user.len() == 0 {
+            colors.colorize("?".to_owned(), &Elem::User)
```

Comment:
as https://github.com/Peltoche/lsd/pull/484#discussion_r575677675 mentioned, this should be `-`, right?

@meain